packages/fiori/src/TimelineItem.hbs

```diff
@@ -10,22 +8,24 @@
 			{{/if}}
 		</div>
 	</div>
-	<div class="ui5-tli-bubble" tabindex="{{forcedTabIndex}}" data-sap-focus-ref>
-		<div class="ui5-tli-title">
-			{{#if name}}
-				{{> name}}
+	{{#unless hideBubble}}
+		<div class="ui5-tli-bubble" tabindex="{{forcedTabIndex}}" data-sap-focus-ref>
+			<div class="ui5-tli-title">
+				{{#if name}}
+					{{> name}}
+				{{/if}}
+				<span>{{titleText}}</span>
+			</div>
+			<div class="ui5-tli-subtitle">{{subtitleText}}</div>
+
+			{{#if innerHTML}}
```

Comment:
not sure in this check. what will happen if i put html comment inside the component?

Reply:
now, when the default slot has accessor we can replace innerHTML => content.length